spec/handlers/newflow/send_reset_password_email_spec.rb

```diff
@@ -54,5 +54,23 @@ module Newflow
         expect(result).to have_routine_error(:cannot_find_user)
       end
     end
+
+    context 'when no email present and no logged in user' do
+      let(:params) do
+        {
+          forgot_password_form: {
+            email: ''
+          }
+        }
+      end
+
+      let(:subject) do
```

Comment:
please use the rspec explicit subject http://www.betterspecs.org/#subject

Reply:
Fixed.